### ISSUE TITLE
No season or episode info scanned

### ISSUE THREAD

**Issue by TiptoeOkin** [gboudreau/XBMCnfoTVImporter.bundle#94]
I tried to import my nfo files stored by MediaElch.
The tvshow.nfo worked well but nothing else.
Here is the logfile 

[com.plexapp.agents.xbmcnfotv.log.zip](https://github.com/gboudreau/XBMCnfoTVImporter.bundle/files/845764/com.plexapp.agents.xbmcnfotv.log.zip)

anything I can do?

**Comment by TiptoeOkin** [comment]
as requested..

The same occurs to all of my TV show folder, all the same structure

[Fünf Freunde.zip](https://github.com/gboudreau/XBMCnfoTVImporter.bundle/files/847337/Funf.Freunde.zip)

**Comment by SlrG** [comment]
Sadly I can't reproduce the error on my system. I tried to add a possible fix anyway, but I don't know if it will work. Please give the newest version of the agent a try. If it still fails, please post the new log.

**Comment by TiptoeOkin** [comment]
still the same problerm. I am running on an rather old server.
It is a Windows Home Server 1.0 so it is a windows server 2003.
Therefore I am running Plex with Version 2.4.9  
Might that be an issue?
[logfiles.zip](https://github.com/gboudreau/XBMCnfoTVImporter.bundle/files/852503/logfiles.zip)

**Comment by SlrG** [comment]
Yes, the newest versions of the agents only support the newest versions of plex. In June 2016 there was a change in plex, that changed the handling of actors, directors and writers completely. The agents were changed to support that. The old way was removed.

So if you are using such an old server version, it is not going to work sadly. You can download the old versions from before from github, but they might contain other bugs, that have been fixed since then.

It might be a better idea to switch your server software. Personally I'm a huge fan of unRAID. But there are other Linux based servers, which support older hardware too and are still able to run the newest plex versions.
